registry.yml

```diff
@@ -128,6 +128,13 @@ components:
     versions:
       0.5.0:
         version: ^1.1.0
+        
+  aframe-fps-look-controls-component:
+    names: fps-look-controls
+    path: dist/fps-look-controls-component.min.js
+    versions:
+      1.0.1:
+        version: ^1.0.x
```

Comment:
This should probably be:

```
     0.7.0:
        version: ^1.0.x
```

where `0.7.0` would be replaced with the oldest version of A-Frame you've tested with.

Reply:
@donmccurdy I think that's true after A-Frame version 1.0 is there.

Reply:
Hm not sure what you mean, since a-frame 1.0.0 doesn't exist, we probably want this component to work with 0.7.0 and up?

Reply:
my bad, yes your right  `1.0.1:` should indeed be `0.7.0:`  or oldest A-Frame version tested